crates/templates/src/manager.rs

```diff
@@ -181,6 +191,15 @@ impl TemplateManager {
         };
         let id = template.id();
 
+        if !install_template_list.is_empty() && !install_template_list.contains(&id.to_owned()) {
+            let message = format!("Skipping template {}...", id);
```

Comment:
If the user has asked for a specific template, do they need to see skip messages for the other templates?  (Currently you print the skipped ones in the list anyway I think?)

Reply:
I agree and will remove it.